packages/codegen-ui-react/lib/forms/react-form-renderer.ts

```diff
@@ -462,9 +463,17 @@ export abstract class ReactFormTemplateRenderer extends StudioTemplateRenderer<
           ),
         );
       } else {
+        const relatedModelStatements: Statement[] = [];
+        // Build effects to grab nested models off target record for relationships
+        Object.entries(formMetadata.fieldConfigs).forEach(([key, value]) => {
+          if (value.relationship?.type === 'BELONGS_TO' || value.relationship?.type === 'HAS_ONE') {
+            // Flatten statments into 1d array
+            relatedModelStatements.push(...buildGetRelationshipModels(key));
```

Comment:
you are still passing in the field name instead of sanitized name || field name, so this will break if the field name happens to have hyphens, etc.
Also, the other feedback was that it could be great if you put this in the loop already in this file so we are not looping more times than necessary (line 490 I think?)

Reply:
Ah, the field name thing got lost in the rebase I did, ill re-add that back in. For the other concern, we can't really move it down to 490 because this needs to be inside the useEffect declared here.